modules/react/common/lib/theming/useThemedRing.ts

```diff
@@ -1,29 +1,41 @@
-import {CanvasThemePalette, EmotionCanvasTheme, useTheme} from '@workday/canvas-kit-react/common';
-import {colors, CSSProperties, inputColors, statusColors} from '@workday/canvas-kit-react/tokens';
-
-import chroma from 'chroma-js';
+import {
+  CanvasTheme,
+  EmotionCanvasTheme,
+  useTheme,
+  ErrorType,
+} from '@workday/canvas-kit-react/common';
+import {brand} from '@workday/canvas-tokens-web';
+import {colors, CSSProperties, statusColors} from '@workday/canvas-kit-react/tokens';
 
 type paletteSelection = Exclude<keyof EmotionCanvasTheme['canvas']['palette'], 'common'>;
 interface ContrastColors {
   outer?: string;
   inner?: string;
 }
 
-const isAccessible = (foreground: string, background: string = colors.frenchVanilla100) => {
-  return chroma.contrast(foreground, background) >= 3;
-};
-
```

Comment:
we no longer use chroma.contrast and we can't cause of css tokens. Might need to add a way for consumers to change this based on background color